PedroChaparro/loomies-backend
# Test: Add missing user endpoints tests

### Coverages 

| File                | Before | After |
| ------------------- | :----: | ----: |
| controllers/user.go | 17.6%  | 73.1% |

### How can I test the changes? 

1. Run the `go test ./...` command from the `api/` directory. 
2. (Optional 1) Run the `go test ./... -coverprofile=coverage.out` command from the `api/` directory.
3. (Optional 2) Run the `go tool cover --html=coverage.out` command from the `api/` directory to see the coverage HTML file. 
4. (Optional 3) Select the `users.go` file and you should see something like: 

![image](https://user-images.githubusercontent.com/62714297/230120189-96d962aa-811e-4b2a-a7f6-7bc86b96528a.png)
